.github/workflows/create-jira-issue.yml

```diff
@@ -0,0 +1,102 @@
+            echo "::set-output name=message::true"
+          else
+            echo "Actor ${{ github.actor }} is NOT a ${TEAM} team member"
+            echo "::set-output name=message::false"
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_DISPATCH_TOKEN }}
+
+      - name: Create ticket
+        if: ( github.event.action == 'opened' && steps.set-ticket-type.outputs.type != 'Task' ) || ( github.event.action == 'opened' && steps.set-ticket-type.outputs.type == 'Task' && steps.is-team-member.outputs.message == 'false' )
+        uses: tomhjp/gh-action-jira-create@v0.1.3
+        with:
+          project: TFE
+          issuetype: "${{ steps.set-ticket-type.outputs.type }}"
+          summary: "${{ github.event.repository.name }} [${{ steps.set-ticket-type.outputs.type }} #${{ github.event.issue.number || github.event.pull_request.number }}]: ${{ github.event.issue.title || github.event.pull_request.title }}"
+          description: "${{ github.event.issue.body || github.event.pull_request.body }}\n\n_Created in GitHub by ${{ github.actor }}._"
+          # customfield_10089 is "Issue Link", customfield_10371 is "Source" (use JIRA API to retrieve)
+          extraFields: '{ "customfield_10089": "${{ github.event.issue.html_url || github.event.pull_request.html_url }}",
+                          "customfield_10371": { "value": "GitHub" },
+                          "components": [{ "name": "${{ github.event.repository.name }}" }],
+                          "labels": ${{ steps.set-ticket-labels.outputs.labels }}}'
+
+      - name: Search
+        if: github.event.action != 'opened'
+        id: search
+        uses: tomhjp/gh-action-jira-search@v0.2.1
+        with:
+          # cf[10089] is Issue Link (use JIRA API to retrieve)
+          jql: 'issuetype = "${{ steps.set-ticket-type.outputs.type }}" and cf[10089] = "${{ github.event.issue.html_url || github.event.pull_request.html_url }}"'
+
+      - name: Sync comment
+        if: github.event.action == 'created' && steps.search.outputs.issue
+        uses: tomhjp/gh-action-jira-comment@v0.1.0
+        with:
+          issue: ${{ steps.search.outputs.issue }}
+          comment: "${{ github.actor }} ${{ github.event.review.state || 'commented' }}:\n\n${{ github.event.comment.body || github.event.review.body }}\n\n${{ github.event.comment.html_url || github.event.review.html_url }}"
+
+      - name: Close ticket
+        if: ( github.event.action == 'closed' || github.event.action == 'deleted' ) && steps.search.outputs.issue
+        uses: atlassian/gajira-transition@v2.0.1
+        with:
+          issue: ${{ steps.search.outputs.issue }}
+          transition: "Closed"
+
+      - name: Reopen ticket
+        if: github.event.action == 'reopened' && steps.search.outputs.issue
+        uses: atlassian/gajira-transition@v2.0.1
+        with:
+          issue: ${{ steps.search.outputs.issue }}
+          transition: "To Do"
```

Comment:
```suggestion
    uses: hashicorp/terraform-random-tfe-utility/.github/workflows/create-jira-issue@main
    secrets: inherit
```

Reply:
The `TEAM` variable is different, and I changed the workflow to create a `GH Issue` instead of a `Bug`.